### ISSUE TITLE
Some way to see original x,y,z values?

### ISSUE THREAD

**Issue by originalfoo** [algernon-A/BOB#45]
In image below, I've got a specific instance of door marker selected in BOB:

![image](https://user-images.githubusercontent.com/1386719/149391790-37a64bc0-6339-4318-8636-7e5b3484d02d.png)

Problem I'm having is I don't know which floor that marker is on. In the BOB UI, the x,y,z,angle are all 0 for that marker and also the marker on the front of the building, yet they are obviously in different positions on the building.

![image](https://user-images.githubusercontent.com/1386719/149392102-4fea8139-c50b-4c76-9a38-6d7d94e68d10.png)

Would it be possible to get some way to see current x,y,z,a of a prop? Maybe as checkbox item tooltip when hovering the item in the list on the left?

Also, if there's some way to visually indicate the height of the prop more clearly that would be useful for props that have no visible form (eg. door markers, dust markers, parking spaces, etc).

**Comment by algernon-A** [comment]
The 0.9 update included changing the markers to match the actual vertical placement (and height) of the selected item - do you think that it's working well enough for your last para?

I'm thinking that current x,y,z can be done as a tooltip on the relevant slider - do you think that would work, or would you want to see that info elsewhere?

**Comment by originalfoo** [comment]
Yup, it's working better in latest update. Having tooltips on sliders would be great for the occasional need to see more detail.
